components/cache.rst

```diff
@@ -71,14 +91,98 @@ Now you can create, retrieve, update and delete items using this cache pool::
     // remove the cache item
     $cache->deleteItem('stats.num_products');
 
-Advanced Usage
---------------
+For a list of all of the supported adapters, see :doc:`/components/cache/cache_pools`.
+
+Advanced Usage (PSR-6)
+----------------------
 
 .. toctree::
     :glob:
     :maxdepth: 1
 
     cache/*
 
+.. _cache-component-psr16-caching:
+
+Simple Caching (PSR-16)
```

Comment:
I'd move the simple cache part (PSF-16) before the complex cache part (PSR-6). That's the usual practice in docs: simple first and then build up to complex.